massa-bootstrap/src/server.rs

```diff
@@ -269,24 +268,24 @@ impl BootstrapServer<'_> {
                     Some(limit),
                 );
 
-                // check whether incoming peer IP is allowed.
-                if let Err(error_msg) = self.white_black_list.is_ip_allowed(&remote_addr) {
-                    server_binding.close_and_send_error(
-                        error_msg.to_string(),
-                        remote_addr,
-                        move || {},
-                    );
-                    self.massa_metrics.inc_bootstrap_peers_failed();
-                    continue;
-                };
-
                 // the `- 1` is to account for the top-level Arc that is created at the top
                 // of this method. subsequent counts correspond to each `clone` that is passed
                 // into a thread
                 // TODO: If we don't find a way to handle the counting automagically, make
                 //       a dedicated wrapper-type with doc-comments, manual drop impl that
                 //       integrates logging, etc...
                 if Arc::strong_count(&bootstrap_sessions_counter) - 1 < max_bootstraps {
```

Comment:
ok with that but an attacker can take all slots ?

Reply:
He can but he should hard spam but could be possible. Before if he hard spam as he was not taking into account in the limit then he could make the node reach the maximum FD open and make a lot of parts of the node crash